data/mods/Magiclysm/furniture_and_terrain/furniture_plants.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "type": "furniture",
+    "id": "f_moonflower",
+    "name": "Moonflower Bush",
+    "description": "A silver colored vine, whose luminescent flowers will only bloom during night.\n\nAmazingly rare, it grows only in places were heroic deeds took place, or so its believed.",
+    "symbol": "y",
+    "color": "cyan",
+    "looks_like": "f_dandelion",
+    "move_cost_mod": 0,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT", "TINY", "FLAMMABLE_ASH", "NOCOLLIDE", "FLOWER" ],
+    "examine_action": "harvest_furn_nectar",
+    "harvest_by_season": [ { "seasons": [ "spring", "summer", "autumn" ], "entries": [ { "drop": "moon_flowers", "base_num": [ 1, 6 ] } ] } ],
+    "bash": { "str_min": 2, "str_max": 6, "sound": "crunch.", "sound_fail": "whish." }
+  },
+  {
+    "type": "furniture",
+    "id": "f_kämpuch",
+    "name": "Kämpuch",
+    "description": "The tiny and lethal rod flowers of the Kämpuch break the underbrush like small ivory thorns.",
+    "symbol": "i",
+    "color": "cyan",
+    "looks_like": "f_dandelion",
+    "move_cost_mod": 0,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT", "TINY", "FLAMMABLE_ASH", "NOCOLLIDE", "FLOWER" ],
+    "examine_action": "harvest_furn_nectar",
+    "harvest_by_season": [ { "seasons": [ "spring", "summer", "autumn" ], "entries": [ { "drop": "kämpuch", "base_num": [ 1, 4 ] } ] } ],
+    "bash": { "str_min": 2, "str_max": 6, "sound": "crunch.", "sound_fail": "whish." }
+  },
+  {
+    "type": "furniture",
+    "id": "f_olieribos",
+    "name": "Olieribos",
+    "description": "Dark colored leaves and bitter roots, firmly entrenched in old earth.  It can be used to ward against evil, or to attract it.",
+    "symbol": "f",
+    "color": "cyan",
+    "looks_like": "f_dandelion",
+    "move_cost_mod": 0,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT", "TINY", "FLAMMABLE_ASH", "NOCOLLIDE", "FLOWER" ],
+    "examine_action": "harvest_furn_nectar",
+    "harvest_by_season": [ { "seasons": [ "spring", "summer", "autumn" ], "entries": [ { "drop": "olieribos", "base_num": [ 1, 4 ] } ] } ],
+    "bash": { "str_min": 2, "str_max": 6, "sound": "crunch.", "sound_fail": "whish." }
+  },
+  {
+    "type": "furniture",
+    "id": "f_moly",
+    "name": "Moly",
+    "description": "A small herb with milk white flowers and pitch black roots.  A magical plant known since Ancient Greece, it was brought to the new-world by european druids long ago.",
```

Comment:
```suggestion
    "description": "A small herb with milk white flowers and pitch black roots.  A magical plant known since ancient Greece, it was brought to the New World by European druids long ago.",
```